jquery/exercises/domaccess.html

```diff
@@ -32,11 +31,10 @@
 <pre id="solution1" style="display:none;">
 $("img");
 $(".logo");
-$(".media");
-$(".tagline")
-$("a.green");
-$(".blue .topic");
-$("#donate a");
+$(".press-logos");
+$(".opener h1");
+$("p.subtitle");
```

Comment:
Changed this to check for p tags classed with 'subtitle'.  There are also links on the page with the same class.